crates/proposer-client/README.md

```diff
@@ -0,0 +1,50 @@
+# Proposer Client
```

Comment:
nit: I'd put a reference link to this md file in the root README.md

Reply:
Sure, I'll leave it for the next PR.